rhaptos/cnxmlutils/xsl/test/media.cnxml.html

```diff
@@ -337,6 +337,7 @@
         alt='This is a dot plot showing average hours of sleep. The number line is marked in intervals of 1 from 5 to 9. Dots above the line show 1 person reporting 5 hours, 1 with 5.5, 3 with 6, 4 with 6.5, 2 with 7, 2 with 8, and 1 with 9 hours.'
         data-longdesc='m16020_DotPlot_description.html'
```

Comment:
Feel free to push back on this. I don't exactly know off the top of my head where `data-longdesc` comes from. Though I reason to believe it's from the generic `data-` processing instruction.

Reply:
I also asked this myself: Why it needs to be redundant. But I don't know who can decide on this redundancy and if it will break things 😕 There is also a redundancy on the `alt` attribute. For now I only fixed it according to the bug report.